assembly/src/parsers/ast/nodes.rs

```diff
@@ -35,6 +35,8 @@ pub enum Instruction {
     Inv,
     Pow2,
     Exp,
+    ExpImm(Felt),
+    ExpBitLength(u32),
```

Comment:
`u32` is probably an overkill here. Max value for this is 64 - so, at the very least we should make it a `u8`.

Ideally, we should have a type which can go up to at most 64 - but this can be done in the future.

Reply:
fixed